### DISCUSSION TITLE
IPED-search with Python

### DISCUSSION THREAD

**Discussion by renthus** [sepinf-inc/IPED#1895] [General]
in the IPED search bar, when we enter a keyword it brings us the result if found and returns it in the table. I'm trying to do this search with python, using the PyLucene library in the index directory as a search engine. Could you tell me if this is possible with any other Python library? And how to do this?

**Comment by lfcnassif** [comment]
Sorry for the long delay, you can try something like this:
https://github.com/sepinf-inc/IPED/discussions/875
